compiler/quilt/tools/build.py

```diff
@@ -79,14 +80,20 @@ def _get_local_args(node, keys):
 def _is_valid_group(group):
     return isinstance(group, dict) or group is None
 
-def _pythonize_name(name):
-    safename = re.sub('[^A-Za-z0-9]+', '_', name).strip('_')
-
-    if safename and safename[0].isdigit():
-        safename = "n%s" % safename
+def pythonize(name):
```

Comment:
Can you move the renaming here to a separate PR. These changes aren't related to export, are they?

Reply:
Already done. #537.  When that clears, this will go away when I merge Master into this.